modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/InteractListener.java

```diff
@@ -106,23 +125,17 @@ public void onPlayerInteractStick(PlayerInteractEvent event) {
                 return;
             }
             // Player is onboard craft and right clicking
-            float rotation = (float) Math.PI * event.getPlayer().getLocation().getYaw() / 180f;
-
-            float nx = -(float) Math.sin(rotation);
-            float nz = (float) Math.cos(rotation);
-
-            int dx = (Math.abs(nx) >= 0.5 ? 1 : 0) * (int) Math.signum(nx);
-            int dz = (Math.abs(nz) > 0.5 ? 1 : 0) * (int) Math.signum(nz);
-            int dy;
 
+            final Vector direction = player.getLocation().getDirection();
             float p = event.getPlayer().getLocation().getPitch();
 
-            dy = -(Math.abs(p) >= 25 ? 1 : 0) * (int) Math.signum(p);
+            direction.setY(-(Math.abs(p) >= 25 ? 1 : 0) * (int) Math.signum(p));
```

Comment:
For consistency with the rest of the file, should this be shortened to referencing abs directly rather than Math.abs?